src/main/java/com/unascribed/fabrication/support/MixinConfigPlugin.java

```diff
@@ -60,16 +60,14 @@ public class MixinConfigPlugin implements IMixinConfigPlugin {
 	@Override
 	public void onLoad(String mixinPackage) {
 		FabConf.reload();
-		Mixins.registerErrorHandlerClass("com.unascribed.fabrication.support.MixinErrorHandler_THIS_ERROR_HANDLER_IS_FOR_SOFT_FAILURE_IN_FABRICATION_ITSELF_AND_DOES_NOT_IMPLY_FABRICATION_IS_RESPONSIBLE_FOR_THE_BELOW_ERROR");
-		FabLog.warn("Fabrication is about to inject into Mixin to add support for failsoft mixins.");
-		FabLog.warn("THE FOLLOWING WARNINGS ARE NOT AN ERROR AND DO NOT IMPLY FABRICATION IS RESPONSIBLE FOR A CRASH.");
+		Mixins.registerErrorHandlerClass("com.unascribed.fabrication.support.MixinErrorHandler");
```

Comment:
The class is named this because Mixin unconditionally prints "Instancing error handler class CLASSNAME" every time there is a mixin error. The class name needs to stay or the class needs to not be registered with Mixin. I believe it'd be fine to just not register it; mixin error handlers don't do what I was hoping they would anyway, which is why I added the Failsoft system.